FAQ.md

```diff
@@ -77,38 +77,63 @@ Post quality is judged by the amount of effort you put into the creation of your
 ### Q: What are user groups and their privileges?
 **A**: Usergroups are what separates the forum's member by ranking or privileges and consists of:
```

Comment:
While this wasn't part of your changes it should be changed in general. Thanks in advance.

Line 77 "### Q: What are user groups and their privileges?" > "### Q: What are the user groups and their privileges?" Objects such as "user groups" should have "the" before them

Line 78 "**A**: Usergroups are what separates" > "**A**: User groups are what separates" 
Since user groups had been separated in line 77 the same sentence structure should be kept for following lines.